webpack.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   entry: './src/index.js',
   output: {
     library: 'Banana',
-    libraryExport: 'default',
+    libraryExport: '',
```

Comment:
Alternately the `libraryExport` setting can be omitted entirely per https://v4.webpack.js.org/configuration/output/#outputlibraryexport